packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -738,6 +731,101 @@ class TextSelectionOverlay {
   }
 }
 
+/// This widget represents a text selection toolbar.
```

Comment:
Nit: No triple slash on a private class?  Or maybe it doesn't matter.

Reply:
According to https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use--for-public-quality-private-documentation if we plan on making this public in the future it is fine (as long as the documentation is quality). I think this would still be flagged if we make it public because the class fields are not documented.

Reply:
Ah I say keep it triple then.  Thanks for looking that up, I never knew exactly what the policy was.